airmozilla/main/static/main/js/calendar.js

```diff
@@ -31,12 +33,16 @@ $(function() {
                 left: 'prev,next today',
                 center: 'title',
                 right: 'month,agendaWeek,agendaDay'
+            },
+            eventAfterAllRender: function(view) {
+                localStorage.setItem('view', view.name);
             }
         });
     };
     startsOnMonday.on('change', function(){
         localStorage.setItem('startsOnMonday', startsOnMonday.prop('checked')? 1 : 0);
         container.fullCalendar('destroy');
+        view = localStorage.getItem('view');
```

Comment:
What is this for? If it's for keeping the same view when re-recreating the calendar then we'd use `view = container.fullCalendar('getView')` no?